lib/galaxy/tools/imp_exp/__init__.py

```diff
@@ -40,15 +49,15 @@ def cleanup_after_job(self):
         new_history = None
         try:
             archive_dir = jiha.archive_dir
-            model_store = store.get_import_model_store_for_directory(archive_dir, app=self.app, user=user)
+            external_chown(archive_dir, jiha.job.user.system_user_pwent(str(getpass.getuser())),
```

Comment:
```suggestion
            external_chown(archive_dir, jiha.job.user.system_user_pwent(getpass.getuser()),
```
this is already a string, on python2 and python3.